### PR TITLE
refactor: move from cf worker to proxy on fly.io

### PR DESCRIPTION
**Pull request by lucacasonato**
This moves the Cloudflare Worker to ﻿a Deno project using Oak, running on fly.io.

### CODE REVIEW

File: proxy/deps.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export { readerFromStreamReader } from "https://deno.land/std@0.89.0/io/streams.ts";
+export { Application, Router } from "https://deno.land/x/oak/mod.ts";
+export type { Context, RouterContext } from "https://deno.land/x/oak/mod.ts";
```
Comment (ry) [inline proxy/deps.ts:3]: Do you want to use versioned URLs here?

Comment (lucacasonato) [reply proxy/deps.ts:3]: oh whoops

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -23,44 +23,40 @@ jobs:
...
+        run: deno fmt --check
+        working-directory: proxy
+
+      - name: Lint
+        run: deno lint --unstable
+        working-directory: proxy
+
+      - name: Test
+        run: deno test --allow-net
+        working-directory: proxy
```
Comment (ry) [inline .github/workflows/ci.yml:49]: lovely

**Review by ry — APPROVED** [review]
LGTM

**Comment by kt3k** [comment]
Should this be reverted now?

**Comment by crowlKats** [comment]
Ideally it should be moved to deploy, no 🤔?

**Pull request merged**